frontend/app/views/spree/products/_thumbnails.html.erb

```diff
@@ -1,19 +1,10 @@
 <%# no need for thumbnails unless there is more than one image %>
```

Comment:
We can't change the behaviour of this file. We can't have it so users upgrade and the logic for displaying images on the frontend changes (I think it's likely in a ton of stores that this wouldn't be caught before production).